### PR TITLE
Add left padding to chosen worker type in Create Task page.

### LINKED ISSUE #5611: The worker type selector on the `Create Task` page is missing left padding when focused
padding

To repro, change the worker type with the select and don't click anywhere else on the page or press tab a couple times until that element is focused.

### PR DESCRIPTION
**Pull request by ahmedDevs**
Add left padding to the worker type selector in Create Task page.

Issue was causing undesired visual, when selecting a worker type at the Create Task page, the selected item appears glued to the left border.

Github Bug/Issue: Fixes #5611

### CODE REVIEW

**Comment by lotas** [comment]
Hi @ahmedDevs thanks for submitting this pull request!

Can you please check this document: [changelog](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md) on how to generate a changelog? 
We only accept PRs when they contain a changelog file which describes the changes. It can be autogenerated with commands found in the doc.

Thank you!

File: changelog/issue-5611-1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: minor
+reference: issue 5611
+---
+The worker type selection at the create taks page now displays the chosen type with appropriate padding from the left.
```
Comment (lotas) [inline changelog/issue-5611-1.md:5]: @ahmedDevs Thanks for adding those. Is this one a duplicate? 
Looks like we only need one changelog, as this PR is adressing single issue.

Thanks!

Comment (ahmedDevs) [reply changelog/issue-5611-1.md:5]: Yes, this is a duplicate. The desired one is:
changelog/issue-5611.md

Comment (lotas) [reply changelog/issue-5611-1.md:5]: great, can you please remove it? One is enough :)

Comment (ahmedDevs) [reply changelog/issue-5611-1.md]: Done! Thanks for letting me know:)

**Review by lotas — DISMISSED** [review]
Just tested it locally, looks good! 
Will wait for tests to finish and will merge.

Thanks and feel free to search for more tickets ;)

**Pull request merged**